core_types/src/eav/eavi.rs

```diff
@@ -72,8 +72,8 @@ impl fmt::Display for Attribute {
             Attribute::EntryHeader => write!(f, "entry-header"),
             Attribute::Link => write!(f, "link"),
             Attribute::LinkRemove => write!(f, "link_remove"),
-            Attribute::LinkTag(tag) => write!(f, "link__{}", tag),
-            Attribute::RemovedLink(tag) => write!(f, "removed_link__{}", tag),
+            Attribute::LinkTag(link_type) => write!(f, "link__{}", link_type),
```

Comment:
Should it be `LinkType` now? Or is this what actually enables real tags?

Reply:
Yes I think this should definitely be Attribute::Link but only if it is the case that it isn't used already